accessing-hmf-data-through-gcp.md

```diff
@@ -0,0 +1,137 @@
+# Accessing Data Through GCP
+
+## Contents
+* [Overview](#overview)
+* [Logging in to GCP](#logging-in-to-GCP)
+* [Accessing Data](#accessing-data)
+    - [Data request bucket](#data-request-bucket)
+    - [Large files and the Manifest JSON](#large-files-and-the-manifest-json)
+* [GCP Costs](#gcp-costs)
+* [Privacy and Security](#privacy-and-security)
+
+## Overview
+
+This guide is meant to give you the basic steps and knowledge to access the data for which you have been approved via the Google Cloud Platform (GCP). It is not meant to be an exhaustive tutorial on GCP, 
+and we suggest getting background on the key services in GCP via the following tuturials:
+
+* [Google Cloud Console](https://console.cloud.google.com/)
+* [Google Cloud SDK](https://cloud.google.com/sdk)
+* [Google Cloud Storage](https://cloud.google.com/storage)
+* [Google Compute Engine](https://cloud.google.com/compute) 
+ 
+We assume you have already acquired and account, if not please read our [Getting a Google Account](getting-a-Google-account.md) page in this documentation repository. 
+
+This guide will perform all operations via the command line. Please ensure you've installed the Google Cloud SDK.
```

Comment:
We might also want to specify the 'locations' in a way: that the cloud console is actually something different then running on your own machine/server.